control/views.py

```diff
@@ -0,0 +1,12 @@
+from django.http import HttpResponse
+from django.template import loader
+
+from control.forms import ConverterForm
+
+
+def home(request):
+    form = ConverterForm()
```

Comment:
Aqui puedes poner la logica de tu funcion, controlandola con lo que haya en el request:

```
if request.method == 'GET':
    # Creas el form 
    form = ConverterForm()
elif request.method == "POST":
    # Lo coges del request y tendra los datos que hayas submiteado
    form = ConverterForm(request.POST)
```
Al final lo pasas al contexto y ya esta.

Aqui viene mejor explicado:
https://docs.djangoproject.com/en/2.2/topics/forms/#the-view

Cuando tengas los datos, ya puedes hacer operaciones y pasar los resultados al contexto para mostrarlos.

Reply:
El tema es que cuando llamas a una URL, normalmente usas GET. En el `form` lo que haces es especificar que el metodo que vas a utilizar el POST.